input/resources/StructureDefinition-extension-reportingProgram.json

```diff
@@ -45,17 +45,14 @@
         "path" : "Extension.value[x]",
         "type" : [
           {
-            "code" : "CodeableConcept"
+            "code" : "Identifier"
           },
           {
             "code" : "canonical",
-            "targetProfile": ["http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/publishable-manifest-cqfm"]
+            "targetProfile": ["http://hl7.org/fhir/uv/crmi/StructureDefinition/crmi-manifestlibrary"]
```

Comment:
note: since the dependencies haven't been updated, this points to the snapshot still. Does not need to be updated as part of this PR, just adding as a note.